test/cql-pytest/test_aggregate.py

```diff
@@ -76,13 +76,14 @@ def test_count_in_partition(cql, table1):
 # value in v
 def test_count_specific_column(cql, table1):
     p = unique_key_int()
-    stmt = cql.prepare(f"insert into {table1} (p, c, v) values (?, ?, ?)")
-    cql.execute(stmt, [p, 1, 1])
-    cql.execute(stmt, [p, 2, 2])
-    cql.execute(stmt, [p, 3, 3])
-    cql.execute(stmt, [p, 4, None])
+    stmt = cql.prepare(f"insert into {table1} (p, c, v, fli) values (?, ?, ?, ?)")
+    cql.execute(stmt, [p, 1, 1, [1]])
+    cql.execute(stmt, [p, 2, 2, None])
+    cql.execute(stmt, [p, 3, 3, [2, 3]])
+    cql.execute(stmt, [p, 4, None, None])
     assert [(4,)] == list(cql.execute(f"select count(*) from {table1} where p = {p}"))
     assert [(3,)] == list(cql.execute(f"select count(v) from {table1} where p = {p}"))
+    assert [(2,)] == list(cql.execute(f"select count(fli) from {table1} where p = {p}"))
```

Comment:
For the benefit of future readers of this patch, it would be nice to add a comment above this line saying that the count is only 2 because empty lists (null) are supposed to be ignored - and say that it reproduces issue #14198.

Reply:
ok